src/main/java/frc/core/util/PID/PIDTalon.java

```diff
@@ -58,13 +60,16 @@ private void setMasterInverted(boolean isInverted) {
 	}
 
 	public void setFollowers(TalonSRX... followers) {
-		for (TalonSRX follower : followers) {
+		this.followers = new ArrayList<>();
```

Comment:
aqui tem uma maneira de tu não precisar instanciar esse array list, que é convertendo esse teu array pra um list com stream
Ex.:

`this.followers = Arrays.stream(followers). // converte pra stream
                            collect(Collectors.toList()); // coleta os valores como uma lista`

Então acho que seria válido deixar esse set só com essa responsabilidade de setar o this.followers e dps fazer esse foreach em um outro método, tipo configFollowers...

Reply:
adaptei o codigo pra aquela questao que o metodo configFactoryDefault tem que ser antes do set do follower